programs/governance-rewards-client/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+edition = "2021"
+name = "governance-rewards-client"
```

Comment:
I've moved the client into its own folder. I suppose it could have been a feature flag or just left for dead-code to remove. That's probably what I would do in the future.